sdk/storage/azure-storage-blob/out/azure/storage/blob/_blob_client.pyi

```diff
@@ -0,0 +1,59 @@
+# Stubs for azure.storage.blob._blob_client (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from ._deserialize import deserialize_blob_properties, deserialize_blob_stream, get_page_ranges_result
+from ._download import StorageStreamDownloader
+from ._generated import AzureBlobStorage
+from ._generated.models import AppendPositionAccessConditions, BlobHTTPHeaders, BlockList, BlockLookupList, CpkInfo, DeleteSnapshotsOptionType, SequenceNumberAccessConditions, StorageErrorException, UserDelegationKey
+from ._lease import BlobLeaseClient, get_access_conditions
+from ._models import BlobBlock, BlobProperties, BlobSasPermissions, BlobType, ContainerProperties, ContentSettings, PremiumPageBlobTier, SequenceNumberAction, StandardBlobTier
+from ._serialize import get_modify_conditions, get_source_conditions
+from ._shared import encode_base64
+from ._shared.base_client import StorageAccountHostsMixin, parse_connection_str, parse_query
+from ._shared.encryption import generate_blob_encryption_data
+from ._shared.request_handlers import add_metadata_headers, get_length, read_length, validate_and_format_range_headers
+from ._shared.response_handlers import process_storage_error, return_response_headers
+from ._shared.uploads import IterStreamer
+from ._upload_helpers import upload_append_blob, upload_block_blob, upload_page_blob
+from datetime import datetime
+from typing import Any, AnyStr, Dict, IO, Iterable, List, Optional, Tuple, Union
+
+class BlobClient(StorageAccountHostsMixin):
+    container_name: Any = ...
+    blob_name: Any = ...
+    snapshot: Any = ...
+    def __init__(self, account_url: str, container_name: str, blob_name: str, snapshot: Optional[Union[str, Dict[str, Any]]]=..., credential: Optional[Any]=..., **kwargs: Any) -> None: ...
+    @classmethod
+    def from_blob_url(cls: Any, blob_url: str, credential: Optional[Any]=..., snapshot: Optional[Union[str, Dict[str, Any]]]=..., **kwargs: Any) -> BlobClient: ...
+    @classmethod
+    def from_connection_string(cls: Any, conn_str: str, container_name: str, blob_name: str, snapshot: Optional[str]=..., credential: Optional[Any]=..., **kwargs: Any) -> BlobClient: ...
+    def get_account_information(self, **kwargs: Any) -> Dict[str, str]: ...
+    def upload_blob(self, data: Union[Iterable[AnyStr], IO[AnyStr]], blob_type: Union[str, BlobType]=..., length: Optional[int]=..., metadata: Optional[Dict[str, str]]=..., **kwargs: Any) -> Any: ...
```

Comment:
All the blob operations that take CpkInfo should take a string parameter `encryption_scope`